reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/LiteralToUniformReductionOpportunity.java

```diff
@@ -25,18 +30,40 @@
 
   private final Expr literalExpr;
```

Comment:
It would be good to give this the more specific type, ConstantExpr, if possible.

Reply:
Done.